<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>输入窗口</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>371</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>初始化属性</string>
      </property>
      <widget class="QLabel" name="label_PromptInput">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>输入是否有效在此显示</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1代表前5名积分制，0代表前3名积分制</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_m">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>女子项目数</string>
       </property>
       <property name="buddy">
        <cstring>spinBox_w</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_w">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>参赛国家数</string>
       </property>
       <property name="buddy">
        <cstring>spinBox_n</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_n">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditWomenRankRule">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>01001</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditMenRankRule">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10110</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请输入各项目积分规则，积分规则不用空格分隔</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>男子项目数</string>
       </property>
       <property name="buddy">
        <cstring>label_3</cstring>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonInit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>320</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确认初始属性</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>输入规则：
1.各项目和国家数仅最多99项；
2.先确认后再切至输出窗口；
3.比赛信息输入m+w行，每行按排名输入</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>241</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>比赛</string>
      </property>
      <widget class="QPushButton" name="pushButtonNationInfo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>321</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确认国家比赛信息</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditEventResults">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>201</width>
         <height>261</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;7 2 1 8 4&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2 4 10&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;7 1 9 4 8&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4 6 7 8 5&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;8 6 1 &lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;9 2 3 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4 3 6 2 8 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;5 3 2 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;5 6 2 &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;7 4 8 3 1 &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>按比赛项目输入前5/3名</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>输出窗口</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>621</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>排名/项目显示窗口</string>
      </property>
      <widget class="QSpinBox" name="spinBox_modeSelect">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>250</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>230</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>显示模式选择</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonDisplayRank">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>250</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>显示各国排名</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonDisplayProject">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>250</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>显示专项信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonDisplayNation">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>250</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>显示国家信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>国家序号选择</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>230</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>项目序号选择</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_pSelect">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>581</width>
         <height>201</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_nSelect">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>581</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>项目序号0~m-1为m个男子项目，m~w+m-1为w个女子项目
显示模式：1-总分榜    2-男子榜    3-女子榜    4-按国家编号顺序</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>初始化属性和国家参数信息确认后
才可使用排名/项目显示服务</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_PromptOutput">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>330</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>输入是否有效在此显示</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>spinBox_m</tabstop>
  <tabstop>spinBox_w</tabstop>
  <tabstop>spinBox_n</tabstop>
  <tabstop>lineEditMenRankRule</tabstop>
  <tabstop>lineEditWomenRankRule</tabstop>
  <tabstop>pushButtonInit</tabstop>
  <tabstop>pushButtonNationInfo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
